.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,46 @@
+<!--
+Thank you for your Pull Request! Please describe the problem this PR fixes and a summary of the changes made.
+Link to any relevant issues, code snippets, or other PRs.
+
+For trivial changes, this template can be ignored in favor of a short description of the changes.
+-->
+
+## Problem
+
+<!--
+Describe the issue this PR is solving
+-->
+
+**Issue number, if available:**
+
+## Changes
+
+<!--
+Summarize the changes introduced in this PR. This is a good place to call out critical or potentially problematic parts of the change.
+-->
+
+**Corresponding docs PR, if applicable:**
+
+## Validation
+
+<!--
+Describe how changes in this PR have been validated. This may include added or updated unit, integration and/or E2E tests, test workflow runs, or manual verification. If manual verification is the only way changes in this PR have been validated, you will need to write some automated tests before this PR is ready to merge.
+
+For changes to test infra, or non-functional changes, tests are not always required. Instead, you should call out _why_ you think tests are not required here.
+
+If changes affect a GitHub workflow that is not included in the PR checks, include a link to a passing test run of the modified workflow.
+--->
+
+## Checklist
+
+<!--
+These items must be completed before a PR is ready to be merged.
+Feel free to publish a draft PR before these items are complete.
+-->
+
+- [ ] If this PR includes a functional change to the runtime behavior of the code, I have added or updated automated test coverage for this change.
```

Comment:
If it's a checklist, should we remove `if` or move the check to the next line
e.g.

- [ ] This PR includes a functional change to the runtime behavior of the code, I have added or updated automated test coverage for this change.


Or

If this PR includes a functional change to the runtime behavior of the code
- [ ] I have added or updated automated test coverage for this change.

Reply:
I thought about that. The checklist is really just a reminder for folks, I don't care whether they actually check the boxes or not. If it becomes confusing, we can update.